src/citrine/_serialization/properties.py

```diff
@@ -703,32 +703,53 @@ def __init__(self,
                  serializable: bool = True,
                  deserializable: bool = True,
                  default: typing.Optional[dict] = None,
-                 override: bool = False):
+                 override: bool = False,
+                 list_of_pairs: bool = False):
         super().__init__(serialization_path,
                          serializable,
                          deserializable,
                          default,
                          override)
         self.keys_type = keys_type if isinstance(keys_type, Property) else keys_type()
         self.values_type = values_type if isinstance(values_type, Property) else values_type()
+        self.list_of_pairs = list_of_pairs
 
     @property
     def underlying_types(self):
         return dict
 
     @property
     def serialized_types(self):
-        return dict
+        if self.list_of_pairs:
+            return list
+        else:
+            return dict
 
-    def _deserialize(self, value: dict) -> dict:
+    def _deserialize(self, value: typing.Union[dict, list]) -> dict:
         deserialized = dict()
+
+        if type(value) == list:
+            for pair in value:
+                deserialized_key = self.keys_type.deserialize(pair[0])
+                deserialized_value = self.values_type.deserialize(pair[1])
+                deserialized[deserialized_key] = deserialized_value
+            return deserialized
+
         for key, value in value.items():
             deserialized_key = self.keys_type.deserialize(key)
             deserialized_value = self.values_type.deserialize(value)
             deserialized[deserialized_key] = deserialized_value
         return deserialized
 
     def _serialize(self, value: dict) -> dict:
```

Comment:
The return type here isn't right if `list_of_pairs` is true.  I'm not sure what the right way to fix it is -- probably either a Union or just leaving the type off.

Reply:
👍 for `Union`